src/ensembl/src/content/app/entity-viewer/components/ev-sidebar-toolstrip/EVSidebarToolstrip.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import noop from 'lodash/noop';
+
+import {
+  evSidebarToolstripConfig,
+  EVSidebarToolstripItem
+} from './evSidebarToolstripConfig';
+
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+import { Status } from 'src/shared/types/status';
+
+import styles from 'src/shared/components/layout/StandardAppLayout.scss';
+
+export const EVSidebarToolstrip = () => {
+  return (
+    <>
+      {evSidebarToolstripConfig.map((item: EVSidebarToolstripItem) => (
+        <div className={styles.sidebarIcon} key={item.name}>
+          <ImageButton
+            key={item.name}
+            buttonStatus={Status.INACTIVE}
+            description={item.description}
+            onClick={noop}
+            image={item.icon}
+          />
+        </div>
```

Comment:
1) compare this with just defining ImageButton properties directly in this file without having a config file:

```
return (
  <>
    <ImageButton
      buttonStatus={Status.INACTIVE}
      description="Search"
      classNames={{[ Status.INACTIVE ]: styles.sidebarIcon }}
      onClick={noop}
      image={SearchIcon}
    />
    <ImageButton
      buttonStatus={Status.INACTIVE}
      description="Bookmarks"
      classNames={{[ Status.INACTIVE ]: styles.sidebarIcon }}
      onClick={noop}
      image={BookmarkIcon}
    />
    ...
  </>
)
```
won't it be easier than having a separate config file?

2) _(definitely for a different PR)_ Looking at the example above, won't you agree that our refactored ImageButton needs another property to style the ImageButton regardless of its status (like, always make it that big)? Now that I think of it, I believe that's what the DEFAULT status could be (ab)used for; but now we are using it for a different purpose.  Perhaps, instead of the `classNames` property it should have a `className` and a `statusClasses` properties?

Reply:
The first one is done and agree with the second. What do you think @msmanoj and @jyothishnt?

Reply:
Are there any other use cases other than `always make it that big`? 
Although it is annoying to have div around the ImageButton, I feel it will be simpler this way rather than having two different class related props without having many use cases.

Reply:
Hard to tell whether we'll have other use cases, but I would argue that a prop is simpler than a wrapper div; and that unnecessary DOM nodes are always evil.

Reply:
It looks like the default classname is always applied regardless of the status we use:
https://github.com/Ensembl/ensembl-client/blob/82540e9fd2a6cb20d8267ff336067b2a44f6085c/src/ensembl/src/shared/components/image-button/ImageHolder.tsx#L16

I guess that can be used to make the button always big?

Reply:
> It looks like the default classname is always applied regardless of the status we use:

This is being changed in Jyo's PR: https://github.com/Ensembl/ensembl-client/pull/231

We realised that we are confused by the existing statuses, and tried to associate them with predictable semantics:

**SELECTED:** applied when ImageButton has been pressed and stays pressed
**UNSELECTED:** opposite of SELECTED; indicates that the button (which stays pressed when pressed) is released
**DEFAULT:** applied to buttons that do not keep the pressed state, but are released immediately (e.g. BrowserReset button); or to some weird edge cases (e.g. the visibility button in TrackPanelListItem can be selected, unselected, or partially selected; we use the default status for the partially selected variant)
**DISABLED:** indicates that the button cannot be interacted with

Going in that direction means that default status with its associated classes will not be automatically applied to all image buttons, but only to those that are actually in the DEFAULT status.

Reply:
Ah right! In that case it makes more sense to introduce another prop.